pydeconz/models/sensor/air_quality.py

```diff
@@ -20,6 +20,8 @@ class TypedAirQualityState(TypedDict):
         "unhealthy",
         "out of scale",
     ]
+    airqualityco2: int
+    airqualityformaldehyd: int
```

Comment:
This is misspelled, I understand its nothing you can do anything about, I've reported it here https://github.com/dresden-elektronik/deconz-rest-plugin/issues/6810

Reply:
I'm afraid this happened by Denglisch - in german its formaldehyd. I check where this needs to be fixed in the repo.

Reply:
As I wrote, I've already reported it. Proposal: you add a second line with the correctly spelled airqualityformaldehyde: int as well, also in the property you try with both formats prioritising the proper spelling, then both will work

Reply:
This is the last thing then we can merge and make a release

Reply:
> As I wrote, I've already reported it. Proposal: you add a second line with the correctly spelled airqualityformaldehyde: int as well, also in the property you try with both formats prioritising the proper spelling, then both will work

while you were writing I directly adapted. I also changed directly to the new format introduced here [#6673#issuecomment-1475150705](https://github.com/dresden-elektronik/deconz-rest-plugin/pull/6673#issuecomment-1475150705)

```
state/airquality_co2_density
state/airquality_formaldehyde_density
```
hope this is ok - else I can undo.
As the HA PR [#87866](https://github.com/home-assistant/core/pull/87866) is not yet merged and the deconz rest api PR [#6673](https://github.com/dresden-elektronik/deconz-rest-plugin/pull/6673) is also pending, I think we can change as nothing is breaking. The DDF is not yet usable (except you build the rest plugin manually)

Reply:
Great! Kudos to you for being on top of all the changes! <3

I would like for us to wait a couple of days for https://github.com/dresden-elektronik/deconz-rest-plugin/pull/6673 to be merged before merging this one as that is still in flux. Lets aim to have it all merged prior to hass 2023.4 branches off in 10 days.